lib/card/set.rb

```diff
@@ -200,9 +200,11 @@ def write_tmp_file set_pattern, anchors, from_file, seq
         # FIXME - this does not properly handle anchorless sets
         # There are special hacks for *all, but others (like *rstar) will not be found by
         # include_set_modules, which will look for Card::Set::Rstar, not Card::Set::Rstar::Blah
-        
+        # This issue appears to be addressed by making the entries, in modules arrays.
+        # If yes remove this comment.
+
```

Comment:
No real change here.  Just noticing that the comment may be obsolete.  If not, we should finally fix this.  Seems like a necessary part of the recent PR about multi-anchor sets.